augly/video/augmenters/ffmpeg/concat.py

```diff
@@ -30,27 +30,41 @@ def __init__(self, video_paths: List[str], src_video_path_index: int):
             "sample_aspect_ratio", self.width / self.height
         )
 
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def get_command(self, video_path: str, output_path: str) -> List[str]:
         """
         Concatenates multiple videos together
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
         """
-        streams = []
-
-        for stream in self.videos:
-            streams.extend(
-                (
-                    stream.video.filter(
-                        "scale", **{"width": self.width, "height": self.height}
-                    ).filter("setsar", **{"ratio": self.sample_aspect_ratio}),
-                    stream.audio,
-                )
-            )
-
-        return ffmpeg.concat(*streams, v=1, a=1, n=len(streams) // 2), {"vsync": 2}
+        inputs = [["-i", video] for video in self.video_paths]
+        flat_inputs = [element for sublist in inputs for element in sublist]
+        scale_and_sar, maps = "", ""
+        for i in range(len(self.video_paths)):
+            scale_and_sar += f"[{i}:v]scale={self.width}:{self.height}[{i}v],[{i}v]setsar=ratio={self.sample_aspect_ratio}[{i}vf];"
+
+        for i in range(len(self.video_paths)):
+            maps += f"[{i}vf][{i}:a]"
+
+        rest_command = f"concat=n={len(self.video_paths)}:v=1:a=1[v][a]"
+        command = [
```

Comment:
I remember you were having trouble putting this command together; have you verified that it works with a variety of videos, putting the videos in different orders, different numbers of videos, etc?

Reply:
I've tested it with 2 videos and put them in different orders but I will try with more videos now and get back to you.

Reply:
So I've tested with three videos and compare it AugLy and it was the same